pom.xml

```diff
@@ -24,6 +24,7 @@
   </issueManagement>
 
   <modules>
+    <module>buildSrc/maven-shade-ext-bnd-transformer</module>
```

Comment:
As discussed over Slack: 
At the moment, all modules defined in [modules](https://github.com/SonarSource/sonarlint-core/blob/d73add3c7c8db586ede432b2a995966a87da0b8a/pom.xml#L26) section are published to Maven Central as those modules are direct or transitive dependencies needed by clients. 
Also, all those modules generate a javadoc.jar as that is needed for a successful deployment to Maven Central (we discovered this the hard way in [BUILD-4655](https://sonarsource.atlassian.net/browse/BUILD-4655)).

So for the successful release of SLCore, these new modules will also have to generate a javadoc jar (can be a dummy one, eg. [here](https://github.com/SonarSource/sonarlint-core/blob/eadfd469b0bfd77256baee561d0af4110f73774d/backend/embedded/pom.xml#L133)) or define some maven property (is there one?) which will exclude it from deployment.

Reply:
@serhat-yenican-sonarsource has discovered https://github.com/SonarSource/sonarlint-core/blob/403cbb13f4967aa00be8c1e2bce44310279ff02c/its/pom.xml#L20

Reply:
Done